Index: Exercicio 108.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Exercicio 108.py b/Exercicio 108.py
new file mode 100644
--- /dev/null	(date 1779899762446)
+++ b/Exercicio 108.py	(date 1779899762446)
@@ -0,0 +1,4 @@
+num = str(input('Digite o preço R$:'))
+for l in num:
+    resultado = num * 2
+print(f'O dobro {num} é {}')
\ No newline at end of file
